templates/getting_started.html.jinja2

```diff
@@ -68,13 +68,8 @@
     
     $('#presets').on('change', function() {
         var selected = $("#presets option:selected").text();
-        if (selected == "Suggested") $("#output").html("- Suggested: Level Order Randomizer of shorter length that showcases a lot of the randomization options while keeping the seed accessible. Moves are in randomized locations with Low prices. Requires 40 GBs to enter Helm and shutoff the Blast-o-Matic. Good Place to start.")
-        else if (selected == "Season 1 Race Settings") $("#output").html("- Season 1 Race Settings (aka Spike Settings): Level Order Randomizer of medium length that starts you with 3 keys and higher possible Troff N Scoff requirements. All Isles Bananaports are active from the start and Moves are in randomized locations with Medium prices. Requires 69<i>nice</i> GBs to enter Helm and shutoff Blast-o-Matic. Optimized for races.")
-        else if (selected == "Coupled LZR Balanced") $("#output").html("- Coupled LZR Balanced: Coupled (Previous loading zones take you back to where you were before) Loading Zone Randomizer of shorter length. Requires 4 keys to beat and all lobbies/levels are open from the start. Moves are in randomized locations with Medium prices. Helm is locked behind 40 GBs and Blast-o-Matic is off.")
-        else if (selected == "Decoupled LZR Balanced") $("#output").html("- Decoupled LZR Balanced: Decoupled (Previous loading zones take you to a new location from where you were before) Loading Zone Randomizer of shorter length. Requires 3 keys to beat and all lobbies/levels are open from the start. Moves are in randomized locations with Medium prices. Helm is locked behind 40 GBs and Blast-o-Matic is off. Adds a lot more options for loading zones for a different experience than Coupled LZR.")
+        if (selected == "Season 2 Race Settings") $("#output").html("- Season 2 Race Settings: Level Order Randomizer of medium length that starts you with keys 1, 3 and 6 and item randomizer. Plenty of other races are set to give a well-rounded experience accessible for veterans and newcomers alike. Optimized for races.")
```

Comment:
I think better phrasing for the second sentence would be "These settings are designed to give a well-rounded... (etc.)"